_practicalities/intro.md

```diff
@@ -115,5 +159,22 @@ This  this page for more information : url to here for example.
   dependencies depending on the version of Python.
 
 
+# Non recommended mitigation
+
+This is a collection of "mitigation" or "solutions" you will find on the web
+and that you will hear about. This is an attempt to acknowlege them, and
+explain why they can't work and what are their drawbacks before you attempt to
+implement them.
+
+### Use a meta-package.
```

Comment:
I think I'd call this something like 'alternative' rather than 'non-recommended'. It can work, and it will work with older versions of pip than our favoured strategy, it's just a bit messy and inconvenient.